fedimg/utils.py

```diff
@@ -164,3 +164,10 @@ def get_image_name_from_ami_name(image_name, region):
 
     return '-'.join(
         [x for x in [name_vt, region, volume_type, respin] if x])
+
+
+def get_image_name_from_ami_name_for_fedmsg(image_name):
```

Comment:
I don't really understand why we need a new function here? Why do we need to name the ami something different than `Fedora-Cloud-Base-26-20180329.0.x86_64` ?

Reply:
Let's just name the ami `Fedora-Cloud-Base-26-20180329.0.x86_64` too

Reply:
I kept the name similar to how we use to have earlier in AWS. We can change the name to `Fedora-Cloud-Base-26-20180329.0.x86_64` but might need to add a field incase a AMI with the same name is already there.

Also, I think it's better to have the volume name included in the image name ie standard & gp2.

Reply:
yeah - I honestly don't really think people use the name that much. I think there is value in being able to search the same name across all regions and find all AMIs that were made from the same compose. I guess there are probably better ways to do that too